base/src/main/java/proguard/optimize/evaluation/SimpleEnumUseSimplifier.java

```diff
@@ -277,6 +279,15 @@ public void visitConstantInstruction(Clazz clazz, Method method, CodeAttribute c
                                            invokedMethodName,
                                            invokedMethodType);
                 }
+                else if (isObjectCloneCallPoppingSimpleEnumArray(
```

Comment:
Can this also trigger when we have a reguler `Object.clone` call that has nothing to do with Kotlin? We shouldn't apply this replacement in that case then? Could be useful to also add a test for that.